Makefile

```diff
@@ -9,7 +9,7 @@
 VERSION := $(shell git describe)
 RFC_3339 := "+%Y-%m-%dT%H:%M:%SZ"
 COMMIT := $(shell git rev-list -1 HEAD)
-BRANCH := latest
+BRANCH := develop
```

Comment:
I think it's better to remove this part, and add something like `DOCKER_TAG` to instead.
Also add `DOCKER_TAG=master` and `DOCKER_TAG=develop` on AppVeyor yaml file in deploy jobs.
The docker-build command should change to `./ci_scripts/docker-push.sh -t ${DOCKER_TAG} -b`

Reply:
I changed the .appveyor.yaml!